lib/matplotlib/colors.py

```diff
@@ -1347,6 +1373,72 @@ def inverse(self, value):
         return value
 
 
+class BivariateNorm:
+    """
+    Normalize a list of two values corresponding to two 1D normalizers
+    """
+    def __init__(self, norm1=None, norm2=None):
+        """
+        Parameters
+        ----------
+        norm1 :
+            An instance of 1D normalizers
+        norm2 :
+            An instance of 1D normalizers
+        """
+        if norm1 is None:
+            self.norm1 = Normalize()
+        else:
+            self.norm1 = norm1
+        if norm2 is None:
+            self.norm2 = Normalize()
+        else:
+            self.norm2 = norm2
+
+    def __call__(self, values, clip=None):
+        """
+        Parameters
+        ----------
+        values : array-like
+            A list of two values to be normalized
+        clip : list of bools, None, optional
+            A list of two bools corresponding to value in values.
+            If clip is None then clip is set according to corresponding
+            normalizers.
+
+        Returns
+        -------
+        A list of two normalized values according to corresponding 1D
+        normalizers.
+        """
+        if clip is None:
+            clip = [self.norm1.clip, self.norm2.clip]
+
+        temp = np.asarray([self.norm1(values[0], clip=clip[0]),
+                          self.norm2(values[1], clip=clip[1])])
+
+        temp[0] = temp[0] * (256)
+        temp[1] = temp[1] * (256)
+        temp = temp.astype(int)
+        return temp[0] + temp[1] * 256
```

Comment:
Since after normalization every variable ends up between 0 to 1. So here I am changing it to index flattened 1d lut of BivariateColormap by mapping 0-1 to 1 to 256*256. Here I am currently hardcoding 256 as number of rows and columns in lut of 2d colormap. But this should be changed to somehow check `N` attribute of BivariateColormap.